node/src/BaseClient.ts

```diff
@@ -3630,6 +3632,80 @@ export class BaseClient {
         return this.createWritePromise(createXLen(key));
     }
 
+    /**
+     * Returns information about the stream stored at `key`.
+     *
+     * @param key - The key of the stream.
+     * @param fullOptions - If `true`, returns verbose information with a limit of the first 10 PEL entries.
+     * If `number` is specified, returns verbose information limiting the returned PEL entries.
+     * If `0` is specified, returns verbose information with no limit.
+     * @returns A map of detailed stream information for the given `key`. See
+     *     the example for a sample response.
+     * @example
+     * ```typescript
+     * const infoResult = await client.xinfoStream("my_stream");
+     * console.log(infoResult); // Output: {
+     *                          //   length: 2,
+     *                          //   'radix-tree-keys': 1,
+     *                          //   'radix-tree-nodes': 2,
+     *                          //   'last-generated-id': '1719877599564-1',
+     *                          //   'max-deleted-entry-id': '0-0',
+     *                          //   'entries-added': 2,
+     *                          //   'recorded-first-entry-id': '1719877599564-0',
+     *                          //   'first-entry': [ '1719877599564-0', ['some_field", "some_value', ...] ],
+     *                          //   'last-entry': [ '1719877599564-0', ['some_field", "some_value', ...] ],
+     *                          //   groups: 1,
```

Comment:
... and align fields format
```suggestion
     *                          //   'groups': 1,
```

Reply:
quotes are only necessary for keys that have dashes.  This is the output produced by VS Code.